walletlibrary/src/main/java/com/microsoft/walletlibrary/wrapper/OpenIdResponder.kt

```diff
@@ -23,10 +24,16 @@ object OpenIdResponder {
         presentationRequest: PresentationRequest,
         requirement: Requirement
     ) {
-        val presentationResponse = PresentationResponse(presentationRequest)
-        presentationResponse.addRequirements(requirement)
+        val presentationResponses = presentationRequest.getPresentationDefinitions().map { PresentationResponse(presentationRequest, it.id) }
+        for (presentationResponse in presentationResponses) {
+            try {
+                presentationResponse.addRequirements(requirement)
+            } catch (exception: IdInVerifiedIdRequirementDoesNotMatchRequestException) {
+                // This will be thrown if a verified ID couldn't be matched
```

Comment:
Should this just fail silently?

Reply:
I've added a check so it will re-throw if it finds none of the presentation responses can add this requirement